python/tk_desktop/desktop_window.py

```diff
@@ -1247,8 +1258,19 @@ def engine_startup_error(self, error, tb):
                     self.current_project["id"],
                 ),
             )
+        elif six.PY3 and DESKTOP_CANT_CONNECT_RE.search(
+            exception_str.replace("\n", "")
+        ):
+            # We replace \n since regex doesn't parse past a \n
```

Comment:
I had a quick play and I think adding  `[\w\W\s]+` instead of `.*` would probably work for you and remove the need to do this replace. For some reason `.` does not handle new lines.
https://regex101.com/r/Ss2J3p/1

Reply:
Good idea! I'm bad at regex. I've linked to your interactive example in the code too.!